.github/workflows/lint_flake8.yml

```diff
@@ -0,0 +1,23 @@
+name: flake8 Lint
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+      
+jobs:
+  flake8-lint:
+    runs-on: ubuntu-latest
+    name: Lint
+    steps:
+      - name: Check out source repository
+        uses: actions/checkout@v3
+      - name: Set up Python environment
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
```

Comment:
Hi @ahernank and @alimanfoo, the code looks good to me. Just adding some suggestion to configure flake 8 with an extended length of up to 88 characters.

1. name: Configure Flake8

  run: echo "[flake8]\nmax-line-length = 88" >> .flake8